templates/base.html

```diff
@@ -10,6 +10,9 @@
       <link rel="stylesheet" type="text/css" href="{{ SITEURL }}/theme/css/icons.css"/>
       <link rel="stylesheet" type="text/css" href="{{ SITEURL }}/theme/css/styles.css"/>
       <meta charset="utf-8" />
+      {% for title, link in SOCIAL %}
```

Comment:
The only thing I'm confused about is this bit—what's it for?

Reply:
The links in the header are for the case where the site owner does not want to put any visible social media links to their website, but still wants to use the site for IndieAuth login. That includes me. I have not social buttons on [my website](https://masoud.abkenar.net/) and this is by choice. Still, my header contains the github and twitter and gitlab links. That's enough for IndieAuth.